graphql/schemabuilder/pagination.go

```diff
@@ -434,6 +506,34 @@ func indexOfPaginationArgs(argType reflect.Type) int {
 	return -1
 }
 
+func getFuncReturnValue(fn interface{}) reflect.Type {
+	typ := reflect.TypeOf(fn)
+	return typ.Out(0)
+}
+
+func (c *connectionContext) consumeTextFilters(sb *schemaBuilder, m *method, typ reflect.Type) error {
+	c.FilterTextFields = make(map[string]*graphql.Field)
+
+	for name, fn := range m.TextFilterFuncs {
+		funcTyp := getFuncReturnValue(fn)
+		if funcTyp != reflect.TypeOf("") {
```

Comment:
can we make a `var typeOfString = reflect.TypeOf("")`

or just use the "Kind" of the funcTyp?

Reply:
I guess this is not on the request path, so nbd.

Reply:
done